### ISSUE TITLE
Support for Polygon Mesh Processing Library

### ISSUE THREAD

**Issue by LeeBamberTGC** [Dark-Basic-Software-Limited/GameGuruRepo#5184]
A free MIT licensed technology that is well maintained can be used to help MAX generate lower LOD levels of geometry to help with rendering on lower end systems, and for other improvements to performance for more demanding level scenes.

Link to the repo is here: https://www.pmp-library.org/

This is marked as a feature request but tagged with performance as it will have a direct benefit to speed of games if implemented. Feel free to post other ideas this API can be used for inside the MAX engine.

**Comment by earthling45** [comment]
Hi Lee, does this mean we have control over a mesh while building our scenes?

In that i mean, decimate it for lod stages and set the highest lod?

**Comment by LeeBamberTGC** [comment]
@earthling45 Yes, there would be some default LOD ratios based on distance and size of object, but we could expose these so the LOD could be tweaked to suit the object more accurately via the developer properties in the object.
